### DISCUSSION TITLE
【委托】求一个手机上可以阅读浏览科研期刊的软件

### DISCUSSION THREAD

**Discussion by DiamondDiavolo** [share121/inter-knot#4849] [委托]
我记得手机上曾经有一个软件，可以阅读好多科研期刊，当时感觉用不到又给删了，现在有用了😭求求各位神通广大的绳匠帮助

**Comment by Robin-txdy** [comment]
同求，我也需要<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >
